lars-reimann/api-editor
# feat: Identification of Union_type

Closes #486.

### Summary of Changes

See related issue.


### Testing Instructions

Open a terminal in `package-parser`
Run `parse-package  api  -p sklearn  -o .\out\`
Check the `union_type` attribute in `scikit-learn__sklearn__1.0.2__api that are created` in `.\out\`
